test/integration/spec/rtcpeerconnection.js

```diff
@@ -332,15 +326,9 @@ describe('RTCPeerConnection', function() {
         ]);
 
         const [localVideoTrack] = (await makeStream({ video: true, fake: true })).getVideoTracks();
-
-        // NOTE(mroberts): https://bugs.webkit.org/show_bug.cgi?id=174327
-        if (isSafari) {
-          pc1.addTrack(localVideoTrack, stream);
-        } else {
-          pc1.removeStream(stream);
-          stream.addTrack(localVideoTrack);
-          pc1.addStream(stream);
-        }
+        // NOTE(mroberts): https://bugs.webkit.org/show_bug.cgi?id=17432
+        stream.addTrack(localVideoTrack);
+        pc1.addTrack(localVideoTrack, stream);
```

Comment:
Two things:

* The URL is slightly wrong from when it was copied. It should be `https://bugs.webkit.org/show_bug.cgi?id=174327`, i.e., it is missing a trailing `7`.
* Since we aren't testing `addStream`/`removeStream` here anymore, I don't think we need to include a link to this particularly bug here. We _should_ add a test for this bug, though, perhaps elsewhere?

Reply:
@markandrus WDYT about adding a separate file just for testing existing bugs? We could then periodically run them to find out which bugs have been addressed.

Reply:
I don't think it buys us very much. Easier to keep in `test/integration/spec/rtcpeerconnection.js`, IMO.